cmd/slides-to-video-manager/Dockerfile

```
@@ -1,5 +1,16 @@
 FROM ubuntu:20.04 as prod
-RUN apt update && apt install -y ca-certificates
+
+# Install all dependencies needed for all-in-one mode (embedded workers)
+# - ca-certificates: Required for Go TLS connections to Google APIs
+# - imagemagick: Required by pdf-splitter worker to extract slides from PDFs
+# - ffmpeg: Required by image-to-video and concatenate-video workers
+RUN apt update && apt install -y ca-certificates imagemagick ffmpeg
```

Hadolint (2.14.0)

[info] Delete the apt lists (/var/lib/apt/lists) after installing something

(DL3009)